app/templates/components/notification-dropdown.hbs

```diff
@@ -0,0 +1,59 @@
+<i class="mail outline icon">
+</i>
+<div class="floating ui teal circular mini label">20</div>
+<div class="ui wide notification popup bottom left transition ">
+  <div class="ui basic inverted horizontal segments">
+    <div class="ui basic left aligned segment weight-800">
+      <p>Notifications</p>
+    </div>
+    <div class="ui basic right aligned segment weight-400">
+      <a href="#">Mark all as Read</a>
+    </div>
+  </div>
+  <div class="ui fluid link celled selection list">
```

Comment:
Populate the list and the counter from a dummy js object array. (To simulate the server response)
Each notification would be in the format

```js
{
    "title": "New Session Proposal for event1 by  user1",
    "description": "Title of proposal",
    "createdAt": new Date(),
    "isRead": false
}
```

You should display only unread notifications in the dropdown. 
The date can be converted to human readable format using [`ember-moment`](https://github.com/stefanpenner/ember-moment) `moment-from-now` helper already included in the project.